SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -718,139 +718,119 @@ void CFVMFlowSolverBase<V, R>::SetUniformInlet(const CConfig* config, unsigned s
 
-    if (iPoint_Local > -1) {
+      if (iPoint_Local > -1) {
 
-      /*--- We need to store this point's data, so jump to the correct
-       offset in the buffer of data from the restart file and load it. ---*/
+        /*--- We need to store this point's data, so jump to the correct
+        offset in the buffer of data from the restart file and load it. ---*/
 
-      index = counter*Restart_Vars[1] + skipVars;
+        auto index = counter * Restart_Vars[1] + skipVars;
 
-      if (SolutionRestart == nullptr) {
-        for (iVar = 0; iVar < nVar_Restart; iVar++)
-          nodes->SetSolution(iPoint_Local, iVar, Restart_Data[index+iVar]);
-      }
-      else {
-        /*--- Used as buffer, allows defaults for nVar > nVar_Restart. ---*/
-        for (iVar = 0; iVar < nVar_Restart; iVar++)
-          SolutionRestart[iVar] = Restart_Data[index+iVar];
-        nodes->SetSolution(iPoint_Local, SolutionRestart);
-      }
+        if (SolutionRestart == nullptr) {
+          for (auto iVar = 0u; iVar < nVar_Restart; iVar++)
+            nodes->SetSolution(iPoint_Local, iVar, Restart_Data[index+iVar]);
+        }
+        else {
+          /*--- Used as buffer, allows defaults for nVar > nVar_Restart. ---*/
+          for (auto iVar = 0u; iVar < nVar_Restart; iVar++)
+            SolutionRestart[iVar] = Restart_Data[index + iVar];
+          nodes->SetSolution(iPoint_Local, SolutionRestart);
+        }
 
-      /*--- For dynamic meshes, read in and store the
-       grid coordinates and grid velocities for each node. ---*/
+        /*--- For dynamic meshes, read in and store the
+        grid coordinates and grid velocities for each node. ---*/
 
-      if (dynamic_grid && update_geo) {
+        if (dynamic_grid && update_geo) {
 
-        /*--- Read in the next 2 or 3 variables which are the grid velocities ---*/
-        /*--- If we are restarting the solution from a previously computed static calculation (no grid movement) ---*/
-        /*--- the grid velocities are set to 0. This is useful for FSI computations ---*/
+          /*--- Read in the next 2 or 3 variables which are the grid velocities ---*/
+          /*--- If we are restarting the solution from a previously computed static calculation (no grid movement) ---*/
+          /*--- the grid velocities are set to 0. This is useful for FSI computations ---*/
 
-        /*--- Rewind the index to retrieve the Coords. ---*/
-        index = counter*Restart_Vars[1];
-        Coord = &Restart_Data[index];
-
-        su2double GridVel[MAXNDIM] = {0.0};
-        if (!steady_restart) {
-          /*--- Move the index forward to get the grid velocities. ---*/
-          index += skipVars + nVar_Restart + turbVars;
-          for (iDim = 0; iDim < nDim; iDim++) { GridVel[iDim] = Restart_Data[index+iDim]; }
-        }
+          /*--- Rewind the index to retrieve the Coords. ---*/
+          index = counter * Restart_Vars[1];
+          const passivedouble* Coord = &Restart_Data[index];
```

Comment:
```suggestion
          const auto* Coord = &Restart_Data[index];
```

Reply:
... it is my anxiety to use auto for <somehing>double ... but passivedouble's are prob fine